sigma/pipelines/panther/panther_sdyaml_pipeline.py

```diff
@@ -0,0 +1,29 @@
+from sigma.pipelines.common import logsource_windows_process_creation
+from sigma.processing.pipeline import ProcessingItem, ProcessingPipeline
+from sigma.processing.transformations import FieldMappingTransformation
+
+
+def panther_sdyaml_pipeline():
+    return ProcessingPipeline(
+        name="Generic Log Sources to Panther Transformation",
+        # Set of identifiers of backends (from the backends mapping) that are allowed to use this processing pipeline.
+        #   This can be used by frontends like Sigma CLI to warn the user about inappropriate usage.
+        # allowed_backends=frozenset(),
+        # The priority defines the order pipelines are applied. See documentation for common values.
+        # priority=20,
+        items=[
+            ProcessingItem(
+                transformation=FieldMappingTransformation(
+                    {
+                        "CommandLine": "command_line",
+                        "Image": "image",
+                        "ParentCommandLine": "parent_command_line",
+                        "ParentImage": "parent_image",
+                    }
+                ),
+                rule_conditions=[
+                    logsource_windows_process_creation(),

```

Comment:
Should this include `logsource_macos_process_creation()` as well?  Based on the description of the PR: MVP with functionality for converting windows & mac processes rules

Reply:
yes and no, on one hand - current functionality is enough to convert sigma rule to sdyaml
pipelines are responsible for re-mapping fields from sigma names to Panther schema eg. https://app.asana.com/0/1205794742454622/1205796614477693/f, so mac os remapping is coming in upcoming PRs
I've created this PR to be able to merge in smaller chunks, reviewing large PRs is not fun at all